src/Tabs.js

```diff
@@ -34,6 +34,11 @@ class Tabs extends Component {
 
     const scope = `${idgen()}`;
 
+    let indicatorClasses = 'indicator';
+    if (this.props.indicatorClasses) {
+      indicatorClasses += ` ${this.props.indicatorClasses}`;
```

Comment:
let's deconstruct `indicatorClasses` on L33:
`const { children, className, defaultValue, indicatorClasses } = this.props;`

Reply:
Done 👍